src/language-js/printer-estree.js

```diff
@@ -3322,8 +3322,35 @@ function shouldGroupFirstArg(args) {
   );
 }
 
+// Support:
+// pipe
+// pipeP
+// pipeK
+// compose
+// composeFlipped
+// composeP
+// composeK
+// flow
+// flowRight
```

Comment:
Since you listed out all the possibilities anyway, perhaps we should make an array with all names and use that instead of a regex?

Reply:
Works for me